scenemanager/src/main/java/io/pixsight/scenemanager/SceneListener.kt

```diff
@@ -7,18 +7,30 @@ package io.pixsight.scenemanager
 interface SceneListener {
 
     /**
-     * This method is called when the current scene is hidden.
+     * Triggered when the out animation of the given sceneId begin
      *
-     * Note that this method is not called if there is no current scene set
+     * @param sceneId the new scene that is displayed to the user.
+     */
+    fun onSceneHiding(sceneId: Int) = Unit
+
+    /**
+     * Triggered when the out animation of the given sceneId finish
      *
-     * @param sceneId the scene that has been hidden.
+     * @param sceneId the new scene that is displayed to the user.
```

Comment:
This should return the scene that has finish its animation so the hidden scene (the same as onSceneHiding) no? If so, the old doc was correct for this case.

Reply:
Fix